src/Engine/ProtoCore/DSASM/Executable.cs

```diff
@@ -50,6 +50,9 @@ public enum OffsetConstants
         public ProtoCore.DSASM.ProcedureTable[] procedureTable { get; set; }
         public ProtoCore.DSASM.SymbolTable[] runtimeSymbols { get; set; }
 
+        public List<CodeBlock> CodeBlocks { get; set; }
```

Comment:
Right now, the Executable is populated in Core.GenerateExecutable so they have to be public.
Added enhancement task here: http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-6193